rootfs/scheduler/__init__.py

```diff
@@ -1082,17 +1102,33 @@ def _buildpack_readiness_probe(self, controller, delay=30, timeout=5, period_sec
                 'failureThreshold': failure_threshold,
             },
         }
+        return readinessprobe
 
-        # Update only the application container with the health check
-        app_type = controller['spec']['selector']['type']
-        namespace = controller['spec']['selector']['app']
-        container_name = '{}-{}'.format(namespace, app_type)
-        containers = controller['spec']['template']['spec']['containers']
-        for container in containers:
-            if container['name'] == container_name:
-                container.update(readinessprobe)
-
-        return controller
+    '''
+    Applies tcp socket readiness probe to the docker app container only if some port is exposed
+    by the docker image.
+    '''
+    def _default_dockerapp_readiness_probe(self, image, delay=5, timeout=5, period_seconds=5,
+                                           success_threshold=1, failure_threshold=1):
+        port = self._get_port(image)
+        if port is None:
+            return None
```

Comment:
i was confused with liveness probe where probe fail will restart the container but since it is readiness probe it shouldn't effect anything other than updating the endpoints which is not concern here as it not a routable app.But still i think its better not to add any probe.